services/user.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = "",
+        last_name: str = ""

```

Comment:
why you still use empty string instead of `None`

Reply:
Because  test_user_service_create_user fails if i use None in this lines